Dockerfile

```diff
@@ -5,7 +5,9 @@ ARG SYSTEMD="false"
 # IMPORTANT: When updating this please note that stdlib archive/tar pkg is vendored
 ARG GO_VERSION=1.13.15
 ARG DEBIAN_FRONTEND=noninteractive
-ARG VPNKIT_VERSION=0.4.0
+# VPNKIT_VERSION is a git commit hash (Feb 9, 2021).
+# The pre-built image is available for amd64 and arm64.
+ARG VPNKIT_VERSION=7f0eff0dd99b576c5474de53b4454a157c642834
```

Comment:
@djs55 could you tag a release for vpnkit?

Reply:
Sure, let me take a look.

Reply:
Is this suitable: https://github.com/moby/vpnkit/releases/tag/v0.5.0

Reply:
Thanks!!

Reply:
Could you push `:0.5.0` to Docker Hub as well

Reply:
Pushed `vpnkit:0.5.0` to Docker Hub (sorry, I forgot to do this yesterday!)

Reply:
thanks!